woodwork/column_accessor.py

```diff
@@ -435,6 +435,12 @@ def box_plot_dict(self, quantiles=None, include_indices_and_values=True):
             include_indices_and_values=include_indices_and_values,
         )
 
+    @_check_column_schema
+    def validate(self, return_indices=False):
```

Comment:
At the column level, I think this should be `return_index` instead of `return_indices`

Reply:
I think the `return_indices` or `return_index` names make me think we are returning the values of the index where invalid values are found, not necessarily a series of True/False/nan values that indicate the validation status for each row.

I guess I'm not sure what we want here - do we want a list of index values for rows that contain invalid values or a mask column that can be used to select the invalid rows?

Reply:
My only problem with returning a mask is that for users to determine whether they actually have any invalid columns, they need to do a `series.all()` call or something. Returning just the invalid values makes it immediately obvious to users whether they have any invalid rows. We could return `None` if it's valid even. 

If we stick with a mask, I agree that the parameter name might want to use the word mask. If we return just invalid indices, I might want to make it `return_invalid_indices` just to make it super clear what's being returned.

Reply:
@tamargrey I think that's a good idea. Returning invalid values instead of a mask is more helpful and easier to understand. With dask and koalas dataframes, it'd be better to let users decide whether or not to compute or write invalid values to disk. With that being said, I think it'd be more consistent across dataframes to return an empty series if the column is valid.